### ISSUE TITLE
AlignSettingsSection:skip_documentation has wrong default value

### ISSUE THREAD

**Issue by tkopra** [MarketSquare/robotframework-tidy#722]
Robotidy 4.9.0 was supposed to just _allow_ skip formatting documentation in AlignSettingsSection transformer. It however does the skipping now by default. Thus I needed to set AlignSettingsSection:skip_documentation=False to prevent that in my old project.

**Comment by bhirsz** [comment]
Do you use global ``--skip-documentation`` or ``:skip_documentation`` in any other transformer? I still see it as False by default in the source code. But I might omit something, or it could `bleed` from some other place to ``AlignSettingsSection``.

**Comment by tkopra** [comment]
Yes actually using skip-documentation = true in "global" configuration. Idea has been to avoid any other transformations in the Documentation. So in that sense this is logical now, just having impact on any old projects configured in this way.

**Comment by bhirsz** [comment]
I believe from this thread that there is no bug, it's just that old configuration may be affected by 4.9.0 change. Disabling formatting all documentation and only enabling it for one formatter should work.

I will close this feature but I will mention that I'm leaning forward not formatting documentation by default (so the global flag will be not needed) except for formatters like order settings which only move node around and do not touch content. So with this upcoming change you will not need any configuration, default should achieve what you need.
